pkg/query-service/app/logs/v3/enrich_query.go

```diff
@@ -181,6 +182,19 @@ func jsonFilterEnrich(filter v3.FilterItem) v3.FilterItem {
 	return filter
 }
 
+func jsonReplaceField(filter v3.FilterItem, fields map[string]v3.AttributeKey) v3.FilterItem {
+	key, found := strings.CutPrefix(filter.Key.Key, "body.")
```

Comment:
yeah right now it's status code `status.code` .

Till will support the searching the existing attributes that we have already created. I will create an issue for this. This can also come with json parsing form the UI (detail page) as well.

Reply:
https://github.com/SigNoz/signoz/pull/4458#discussion_r1505420199